RGB.NET.Devices.Asus/Generic/AsusUpdateQueue.cs

```diff
@@ -48,22 +48,65 @@ protected override void Update(Dictionary<object, Color> dataSet)
         {
             try
             {
-                foreach (KeyValuePair<object, Color> data in dataSet)
+                if (Device.Type == 0x00080000 || Device.Type == 0x00081000)//Keyboard
                 {
-                    int index = (int)data.Key;
-                    IAuraRgbLight light = Device.Lights[index];
-                    (_, byte r, byte g, byte b) = data.Value.GetRGBBytes();
-                    light.Red = r;
-                    light.Green = g;
-                    light.Blue = b;
+                    foreach (KeyValuePair<object, Color> data in dataSet)
+                    {
+                        ushort index = (ushort)data.Key;
+                        IAuraSyncKeyboard keyboard = (IAuraSyncKeyboard)Device;
+                        if (keyboard != null)
+                        {
+                            IAuraRgbLight light;
+                            //UK keyboard Layout
+                            if (index == 0x56)
```

Comment:
Hey, could you add some information what all these magic numbers come from and what exactly they mean?